Proofs/SHA512/SHA512Sym.lean

```diff
@@ -47,6 +47,7 @@ abbrev ktbl_addr : BitVec 64 := 0x1b4300#64
 -- set_option profiler true in
 -- set_option profiler.threshold 1 in
 -- set_option pp.deepTerms false in
+set_option trace.Tactic.cse.summary true in
```

Comment:
I'm wary of this -- there's a lot of `make` output already from `simp_mem`. Could you tell me what the output here looks like?

Reply:
Also, if there's no `#guard_msgs` here, I'd rather this was turned off.

Reply:
Agreed.

Reply:
I removed the trace.